File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0311Q00LayRootListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0311Q00LayRootListInfo
    {
        private String _setPart;
        private String _hangmogCode;
        private String _hangmogName;

        public String SetPart
        {
            get { return this._setPart; }
            set { this._setPart = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public OCS0311Q00LayRootListInfo() { }

        public OCS0311Q00LayRootListInfo(String setPart, String hangmogCode, String hangmogName)
        {
            this._setPart = setPart;
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0311Q00LayRootListInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0311Q00LayRootListInfo {
	private String setPart       ;
	private String hangmogCode   ;
	private String hangmogName   ;
	public OCS0311Q00LayRootListInfo(String setPart, String hangmogCode,
			String hangmogName) {
		super();
		this.setPart = setPart;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
	}
	public String getSetPart() {
		return setPart;
	}
	public void setSetPart(String setPart) {
		this.setPart = setPart;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	

}
